lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/service/AssignmentEntity.java

```diff
@@ -87,6 +87,8 @@ public class AssignmentEntity implements LessonEntity, AssignmentInterface {
     public static final int CACHE_TIME_TO_LIVE_SECONDS = 600;
     public static final int CACHE_TIME_TO_IDLE_SECONDS = 360;
 
+    public static final int NON_ELECTRONIC_ASSIGNMENT_SUBMISSION = 4;
```

Comment:
I'd use Assignment.SubmissionType.NON_ELECTRONIC_ASSIGNMENT_SUBMISSION.ordinal() here. I know it's more wordy, but I think this is the original and if it changes, you need to change this lessons specific one.

Reply:
Good idea, thanks!